arrays/practiceObjectOriented/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+type Rectangle struct {
+	length int
+	width  int
+}
+
+func Monolithic(length, width int) (int, int) {
+	var r Rectangle
+	r.initialize(length, width)
+
+	area := r.findArea()
+	peri := r.findPerimeter()
+	return area, peri
+}
+
+func (r *Rectangle) findArea() int {
```

Comment:
as discussed remove the pointers as it is not required

Reply:
fixed